repository_wasi_test.go

```diff
@@ -0,0 +1,35 @@
+//go:build wasip1
+// +build wasip1
+
+package git
+
+import (
+	"testing"
+
+	"github.com/go-git/go-billy/v6/memfs"
+	"github.com/go-git/go-git/v6/storage/memory"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func preReceiveHook(_ string) []byte {
+	return []byte{}
+}
```

Comment:
This needs to be moved into `worktree_wasi.go` as this is required for the wasi build tag to work.